src/SFA.DAS.ProviderApprenticeshipsService.Web/Orchestrators/DataLockOrchestrator.cs

```diff
@@ -12,13 +12,12 @@
 using SFA.DAS.ProviderApprenticeshipsService.Application.Queries.GetApprenticeshipPriceHistory;
 using SFA.DAS.ProviderApprenticeshipsService.Application.Queries.GetCommitment;
 using SFA.DAS.ProviderApprenticeshipsService.Domain.Interfaces;
-using SFA.DAS.ProviderApprenticeshipsService.Web.Exceptions;
 using SFA.DAS.ProviderApprenticeshipsService.Web.Models.DataLock;
 using SFA.DAS.ProviderApprenticeshipsService.Web.Orchestrators.Mappers;
 
 namespace SFA.DAS.ProviderApprenticeshipsService.Web.Orchestrators
 {
-    public sealed class DataLockOrchestrator
+    public sealed class DataLockOrchestrator : BaseCommitmentOrchestrator
     {
         private readonly IMediator _mediator;
```

Comment:
No need for _mediator, _hashingService or _logger in this class anymore.
Instead we can use Mediator, HashingService and Logger from the base class BaseCommitmentOrchestrator instead.